src/main/java/ru/javarush/quest/entity/Account.java

```diff
@@ -0,0 +1,64 @@
+package ru.javarush.quest.entity;
+
+public class Account {
```

Comment:
You have already added lombok. Why not use it for toString, equals and hashCode methods generation

Reply:
I deleted lombok. I dont use already.